### PR TITLE
Devcontainer Update

### PR DESCRIPTION
**Pull request by bartr**
## Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [x] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

### Purpose of PR
- added devcontainer support
- added WebV test files

### Validation
- n/a Unit tests updated and ran successfully
- n/a Update documentation or issue referenced above

### Issues Closed or Referenced

- References retaildevcrews/helium#396

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "helium-typescript",
+
+  "settings": { "terminal.integrated.shell.linux": "/bin/bash" },
+
+	// Open port 4120 by default
+  "forwardPorts": [4120],
+
+  // Install extensions
```
Comment (evillgenius75) [inline .devcontainer/devcontainer.json:9]: Are these the best extensions for a JS/TS dev experience?

Comment (aflinchb) [reply .devcontainer/devcontainer.json:9]: I hadn't added any special extension for JS/TS, but we probably do not need the ms-dotnettools.csharp

File: docs/vscode-template/launch.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
```
Comment (aflinchb) [inline docs/vscode-template/launch.json:12]: Could you also remove our .vscode folder (and add to .gitignore)? If not, I'll get it in my next PR.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (bartr) [reply .devcontainer/devcontainer.json:9]: I removed it, but, for some reason I think it's needed. If so, we'll add back and PR.

File: docs/vscode-template/launch.json (hunk 2 above)
Comment (bartr) [reply docs/vscode-template/launch.json:12]: good catch - removed

**Pull request merged**